bouncer/shared/contract_swap.ts

```diff
@@ -117,10 +118,12 @@ export async function performSwapViaContract(
     swapContext?.updateStatus(swapTag, SwapStatus.ContractApproved);
 
     const oldBalance = await getBalance(destAsset, destAddress);
-    console.log(`${tag} Old balance: ${oldBalance}`);
-    console.log(
-      `${tag} Executing (${sourceAsset}) contract swap to(${destAsset}) ${destAddress}. Current balance: ${oldBalance}`,
-    );
+    if (log) {
+      console.log(`${tag} Old balance: ${oldBalance}`);
+      console.log(
+        `${tag} Executing (${sourceAsset}) contract swap to(${destAsset}) ${destAddress}. Current balance: ${oldBalance}`,
+      );
+    }
```

Comment:
why would we not want to log stuff like this? Feel like we should always log and simplify the logic, remove the condition

Reply:
I needed to run the swapping test at the same time as my FoK test to trigger an error i was getting, but the logs were getting overwhelmed by the amount of logs from the swaps, making it had to see the logs from my FoK test, so i needed a way to disable the logs that i didn't care about. thought no harm in leaving it in, same as the non-contract swaps have this log condition that is on by default.

Reply:
Not a strong opinion so I'm ok with leaving it, but with the tags shouldn't it be relatively easy to trace the logs from a particular test?